src/module-elasticsuite-catalog/etc/frontend/di.xml

```diff
@@ -249,4 +249,11 @@
         <!-- This plugin is defined in magento/module-inventory-catalog -->
         <plugin name="update_toolbar_count" disabled="true"/>
```

Comment:
@romainruaud this should be removed:
[2023-04-14T10:34:05.080956+00:00] report.INFO: Reference to undeclared plugin with name 'update_toolbar_count'. [] []